docs/src/pages/overview/Overview.jsx

```diff
@@ -125,125 +125,123 @@ function CustomThemes() {
 
       <SyntaxHighlighter language="javascript" style={docco}>
         {`
-  import { DxcButton, ThemeProvider } from "@dxc-technology/halstack-react";
+          import { DxcButton, ThemeProvider } from "@dxc-technology/halstack-react";
```

Comment:
I think we should remove the first tab, looks very alienated on the right.

Reply:
Done!